set_version

```diff
@@ -41,10 +41,28 @@ Usage: $0 FLAGS...
   --file FILE:                                     Modify another file than ${FILE}, useful if run
                                                    outside of the SDK chroot. If /dev/stdout or
                                                    /dev/stderr is used, only new values are printed.
+  --binhost                                        Use a custom binhost (defaults to '${DEFAULT_BASE_URL}').
+                                                   This will update BOARD and SDK URLs accordingly.
 "
   exit 1
 fi
 
+# --binhost needs to be evaluated first since it impacts other variables set below
+custom_binhost="false"
+for arg in ${@} ; do
+    if $custom_binhost; then
+        echo "Using custom binhost '${arg}'"
+        DEFAULT_BASE_URL="${arg}"
+        DEV_BOARD_URL="${DEFAULT_BASE_URL}/developer"
+        DEFAULT_SDK_URL="${DEFAULT_BASE_URL}/sdk"
+        DEV_SDK_URL="${DEFAULT_BASE_URL}/developer/sdk"
+        break
+    fi
+    if [ "$arg" = "--binhost" ] ; then
```

Comment:
You will need to add this to `case` below, otherwise won't you get the typoed error `"Unkown argument --binhost"`?

Reply:
Good catch! Will fix.